src/Skoruba.IdentityServer4.Admin/Program.cs

```diff
@@ -8,13 +9,24 @@ namespace Skoruba.IdentityServer4.Admin
 {
     public class Program
     {
+        private const string SeedArgs = "/seed";
+
         public static async Task Main(string[] args)
         {
+            var seed = args.Any(x => x == SeedArgs);
+            if (seed) args = args.Except(new[] { SeedArgs }).ToArray();
+
             var host = BuildWebHost(args);
 
             //NOTE: Uncomment the line below to use seed data
             //await DbMigrationHelpers.EnsureSeedData(host);
 
+            //Eventually use seed via command - dotnet run /seed
```

Comment:
This comments says that eventually you will receive the arg, but to me it looks like this change makes it so that that's the case now. Is the comment still accurate?

Reply:
I meant for seeding you can uncomment line 22 or use command `dotnet run /seed`. 
You are right it isn't probably clear. Can you help me with better comment? Maybe remove lines 21,22 and use only seeding via args?

Reply:
I see what you mean now, so maybe this would work as a replacement for 21-28:

```cs
// Uncomment this to seed upon startup, alternatively pass in `dotnet run /seed` to seed using CLI
//await DbMigrationHelpers.EnsureSeedData(host);
if (seed)
{
    await DbMigrationHelpers.EnsureSeedData(host);
}
```

This way it's colocated and makes more sense to me, in my original comment I didn't notice the two blocks of code were related before taking another look after your comment.

Reply:
It is great, thanks! :-)